### PR TITLE
rMVP analytics

### PR DESCRIPTION
**Pull request by jmelvnsn**
@collin still need to add tests

### CODE REVIEW

File: elements/bulbs-video/components/revealed.js [hunk 1]
```diff
@@ -269,6 +245,14 @@ export default class Revealed extends React.Component {
     GoogleAnalytics.init(player, videoMeta.gaPrefix);
     Comscore.init(player, global.BULBS_ELEMENTS_COMSCORE_ID, videoMeta.player_options.comscore.metadata);
 
+    if (this.props.onPlay) {
```
Comment (collin) [inline elements/bulbs-video/components/revealed.js]: We should ask @spra85 about how he tested the jw events for the GoogleAnalytics plugin above.

File: elements/bulbs-video/elements/rail-player/components/root.js [hunk 2]
```diff
@@ -1,24 +1,45 @@
...
 
+import {getAnalyticsManager} from 'bulbs-elements/util';
+
 import Revealed from '../../../components/revealed';
 
 import RailPlayerHeader from './header';
 import RailPlayerCampaign from './campaign';
 
 export default class Root extends React.Component {
+
```
Comment (collin) [inline elements/bulbs-video/elements/rail-player/components/root.js]: These will be reasonably straightforward to test.

File: lib/bulbs-elements/util/get-analytics-manager.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        1. Have bulbs-video and/or videohub-player initialize their own
+           analytics manager
+        2. Have bulbs-video and/or videohub-player use a confuration
+           such as BULBS_ELEMENTS_ANALYTICS_MANAGER.
+        3. Have all sites follow a convention for where AnalyticsManager
+           lives.
+*/
+/* global avclubAnalytics, onionan, clickholean, starwipe */
+
+export default function() {
```
Comment (collin) [inline lib/bulbs-elements/util/get-analytics-manager.js]: As will this.

File: elements/bulbs-video/components/revealed.js (hunk 1 above)
Comment (spra85) [inline elements/bulbs-video/components/revealed.js]: I know we needed the click tracking elsewhere on the rMVP, but aren't play and pause already tracked by the default implementation?

Comment (collin) [inline elements/bulbs-video/components/revealed.js]: Yes, but that is not using the same semantics as our clicktracking system, might need to coordinate with Rachel/Kelly to see if they can re-use that tracking.

File: lib/bulbs-elements/util/get-analytics-manager.js (hunk 3 above)
Comment (jmelvnsn) [inline lib/bulbs-elements/util/get-analytics-manager.js]: @collin https://github.com/theonion/bulbs-elements/pull/152/commits/83bf7593424e39b278605ec6ea90f2cd4eb5eb45

**Comment by collin** [comment]
👍

**Pull request merged**